pokemongo_bot/human_behaviour.py

```diff
@@ -24,6 +24,9 @@ def random_lat_long_delta():
     # Return random value from [-.000025, .000025]. Since 364,000 feet is equivalent to one degree of latitude, this
     # should be 364,000 * .000025 = 9.1. So it returns between [-9.1, 9.1]
     return ((random() * 0.00001) - 0.000005) * 5
+    
+def random_lat_long_delta2(bearing):
+    return ((random() * 0.00001) - 0.000005) * 5, ((random() * 0.00001) - 0.000005) * 5
```

Comment:
If we're heading at 0/360, we only add variance to latitude. Do some more math between there to scale each respective to that. I'm not sure